Dolphin/windWakerInterface.py

```diff
@@ -1,57 +1,57 @@
 import random
-
 import dolphin_memory_engine as dme
-
+from typing import List
 import Dolphin.windWakerResources as WWR
 from Model.config import Config
 
 random_rupoors = Config.get_config().Random_Rupoors
 
-def hook():
+
+def hook() -> None:
     dme.hook()
 
 
-def assert_hook():
+def assert_hook() -> None:
     dme.assert_hooked()
 
 
-def is_hooked():
+def is_hooked() -> bool:
     return dme.is_hooked()
 
 
 def write_byte(address, value: int) -> None:
     dme.write_byte(address, value)
 
 
-def read_byte(address):
+def read_byte(address) -> None:
     return dme.read_byte(address)
 
 
-def read_word(address):
+def read_word(address) -> int:
     return dme.read_word(address)
 
 
-def write_word(address, value):
+def write_word(address, value) -> None:
     return dme.write_word(address, value)
 
 
-def write_float(address, value) -> None:
+def write_float(address, value: float) -> None:
     dme.write_float(address, value)
 
 
-def read_float(address):
+def read_float(address) -> float:
     return dme.read_float(address)
 
 
-def read_chest_items():
+def read_chest_items() -> List[int]:
     return [(0xFF & read_word(0x803FED40)), (0xFF & read_word(0x803FED44))]
 
 
-def clear_chest_items():
+def clear_chest_items() -> List[None]:
```

Comment:
The best part of these reviews is seeing my own code again and thinking "Why in the world did I write it like this"

Nothing wrong with your PR here, just felt like putting in something funny. Makes reviews fun instead of a chore.